bridge-api/definitions/paged_resources/forward_cursor_scheduled_activity.yml

```diff
@@ -7,7 +7,7 @@ properties:
         items:
             $ref: ../scheduled_activity.yml
     offsetBy:
-        type: number
+        type: string
```

Comment:
These are rolled together, but since swagger.json is  not directly used in Java SDK, this should not be a problem. Both are going in today probably.